migrate/migrate.go

```diff
@@ -25,18 +25,24 @@ import (
 
 	"github.com/dapperlabs/flow-playground-api/controller"
 	"github.com/dapperlabs/flow-playground-api/model"
+	"github.com/dapperlabs/flow-playground-api/storage"
 )
 
 type Migrator struct {
 	projects *controller.Projects
+	store    storage.Store
 }
 
 var V0 = semver.MustParse("v0.0.0")
 var V0_1_0 = semver.MustParse("v0.1.0")
 
-func NewMigrator(projects *controller.Projects) *Migrator {
+// for migration to new seperate Contract struct re:flipfest-21
+var V0_8_0 = semver.MustParse("v0.8.0")
```

Comment:
Yes `v0.8.0` is most likely the release version this will have. I think the original Idea was to have this match the release version. I will double check.

Reply:
yes